lib/browser/api/app.js

```diff
@@ -77,6 +79,28 @@ app.allowNTLMCredentialsForAllDomains = function (allow) {
   }
 }
 
+const makeSingleInstance = app.makeSingleInstance
+app.makeSingleInstance = function (callback) {
+  return makeSingleInstance.call(app, function (argv, ...args) {
+    if (process.platform === 'win32') {
+      // Delay the checking so the callback is run as quickly as we would expect
+      setImmediate(() => {
+        argv.forEach((arg) => {
```

Comment:
Handling the first argument should be enough, Explorer and other shells are only passing one URL at one time.